gooddata-fdw/sonar-project.properties

```diff
@@ -0,0 +1,2 @@
+sonar.projectKey=gooddata_python-sdk_gooddata-fdw
```

Comment:
Add also keys:
- `sonar.projectName` - https://docs.sonarcloud.io/advanced-setup/analysis-parameters/
- `sonar.python.version=3.7, 3.8, 3.9, 3.10`
- `sonar.sources` - https://docs.sonarcloud.io/advanced-setup/analysis-scope/
- `sonar.tests` - https://docs.sonarcloud.io/advanced-setup/analysis-scope/

Reply:
Added to other projects too.